<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppCfgDialogClass</class>
 <widget class="QDialog" name="AppCfgDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppCfgDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="ElaPushButton" name="pushButton_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="ElaText" name="label">
     <property name="text">
      <string>探测器电量阈值</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ElaPushButton" name="pushButton_confirm">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ElaSpinBox" name="spinBox_detLowBattery"/>
   </item>
   <item row="2" column="1">
    <widget class="ElaCheckBox" name="checkBox_autoStartXRay">
     <property name="text">
      <string>采集前自动开启射线</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="ElaCheckBox" name="checkBox_autoStopXRay">
     <property name="text">
      <string>采集后自动关闭射线</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ElaPushButton</class>
   <extends>QPushButton</extends>
   <header>ElaPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ElaText</class>
   <extends>QLabel</extends>
   <header>ElaText.h</header>
  </customwidget>
  <customwidget>
   <class>ElaSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ElaSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ElaCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ElaCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
